### PR TITLE
cancelled deploys should clear deploymentId

### PR DESCRIPTION
**Pull request by sprkem**
*Issue #, if available:*

*Description of changes:*

When cancelling a deploy we were not properly clearing the deployment marker on a MFE which prevented subsequent deployments.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: infrastructure/lambda/adminApi/app.js [hunk 1]
```diff
@@ -495,7 +495,7 @@ export const deleteDeploymentApi = middy().handler(async (event, context) => {
       projectId: projectId,
       microFrontendId: microFrontendId,
     },
-    UpdateExpression: "set activeVersions = :v, #def = :d",
+    UpdateExpression: "set activeVersions = :v, #def = :d remove deploymentId",
```
Comment (sprkem) [inline infrastructure/lambda/adminApi/app.js:498]: This might look odd with no comma but it's correct!

Comment (lucamezzalira) [reply infrastructure/lambda/adminApi/app.js:498]: shouldn't ```remove``` being all capital latters?

Comment (sprkem) [reply infrastructure/lambda/adminApi/app.js:498]: It works the same. See the pre-existing set is lower case also.

**Review by matteofigus — APPROVED** [review]
Should we add a integration test to ensure this doesn’t happen in the future?

**Pull request merged**